glyphs-reader/src/font.rs

```diff
@@ -1397,6 +1408,10 @@ impl RawFeature {
             ))),
         }
     }
+
+    fn disabled(&self) -> bool {
+        self.disabled.map(|v| v == 1).unwrap_or_default()
```

Comment:
`self.disabled == Some(1)` looks a bit more clear to me.

Reply:
Agreed, updated